ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -325,6 +376,18 @@ func deleteCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	return nil
 }
+func deleteEmptyClusterPrompt(reader *bufio.Reader, cliParams *config.CLIParams) error {
```

Comment:
`deleteEmptyClusterPrompt` and `deleteClusterPrompt` are very similar functions. I'd suggest potentially refactoring them to be one function that takes a prompt. Or even better might be to have one function which reads an input line and returns true if it was y/yes and false otherwise. This would also help if you take my suggestion to refactor `down` so that the user never has to confirm more than once.